packages/module/src/DataViewTable/DataViewTable.tsx

```diff
@@ -1,57 +1,94 @@
-import { FC, ReactNode } from 'react';
-import {
-  TdProps,
-  ThProps,
-  TrProps
-} from '@patternfly/react-table';
+import { FC, ReactNode, MouseEvent as ReactMouseEvent, KeyboardEvent as ReactKeyboardEvent } from 'react';
+import { TdProps, ThProps, TrProps, InnerScrollContainer } from '@patternfly/react-table';
 import { DataViewTableTree, DataViewTableTreeProps } from '../DataViewTableTree';
 import { DataViewTableBasic, DataViewTableBasicProps } from '../DataViewTableBasic';
 
+// Table resizable typings
+export interface DataViewThResizableProps {
+  /** Whether the column is resizable */
+  isResizable?: boolean;
+  /** Callback after the column is resized */
+  onResize?: (
+    event: ReactMouseEvent | MouseEvent | ReactKeyboardEvent | KeyboardEvent | TouchEvent,
+    width: number
+  ) => void;
+  /** Width of the column */
+  width?: number;
+  /** Minimum width of the column */
+  minWidth?: number;
+  /** Increment for keyboard navigation */
+  increment?: number;
```

Comment:
Not a blocker, more thinking out loud. What do you think about being able to use the Shift key to alter the increment amount? I know with Photopea (and probably other graphic design programs), moving layers with the arrow key moves it X pixels, but holding shift moves it Y pixels (I think 1 and 10 respectively?).

We can wait to see if anyone requests more fine tuned control of the increment amount, but wanted to mention it in case there's anything we could do now to make it easier to implement should that ever arise (though maybe it's just another prop in the future and we check for the shift key in the existing logic?)